OpenProblemLibrary/AlfredUniv/anton8e/chapter1/1.5/prob9.pg

```diff
@@ -44,7 +44,7 @@ TEXT(beginproblem());
 
 $ans1 = Formula("1/sqrt(x**2 + 1)")->reduce;
 $ans2 = Formula("sqrt(1-1/x**2)")->reduce;
-$ans2->with(limits=>[1.1, 10], test_at=>[[-2],[-3]]);
+$ans2 = $ans2->with(limits=>[1.1, 10], test_at=>[[-2],[-3]]);
```

Comment:
```suggestion
$ans2->{limits}=[1.1, 10];
$ans2->{test_at}=[[-2],[-3]];
```
I believe that this is the preferred syntax.

This problem also needs
```suggestion
Context()->functions->disable('Trig');
```
Otherwise a student can simply enter cos(arctan(x)) and get credit for the problem.